src/mlpack/bindings/go/CMakeLists.txt

```
@@ -9,6 +9,30 @@ macro (not_found_return message)
   endif()
 endmacro ()
 
+macro (post_go_setup)
+  if (BUILD_GO_BINDINGS)
+    # If we are building Go bindings, we have to end the 'module' declaration in
+    # models.go
```

```suggestion
    # Once `GO_MODELS` is populated, generate `models.go`.
```

Is this right?  I don't think that that there is any `module` that needs to be closed.  @Yashwants19, I think you originally wrote this; does my change look okay?  Let me know what you think. :+1:

---

This is looking awesome, Thanks for improving the documentation. :)